sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/api/Microsoft.Azure.WebJobs.Extensions.Tables.netstandard2.0.cs

```diff
@@ -42,18 +22,6 @@ public partial class TableAttribute : System.Attribute, Microsoft.Azure.WebJobs.
         public int Take { get { throw null; } set { } }
```

Comment:
```suggestion
        public int MaxPerPage { get { throw null; } set { } }
```

Reply:
We are keeping attributes as compatible with the previous version as possible.